geode-core/src/integrationTest/java/org/apache/geode/internal/io/ServerJdkGlobalSerialFilterIntegrationTest.java

```diff
@@ -64,6 +65,8 @@ public void setUp() throws IOException {
   }
 
   @Test
+  @Ignore
+  // jdk.serialFilter does not apply to ServerLauncher
```

Comment:
This test may not be finished.

ServerLauncher should NOT configure global serial filter on Java 8 or Java 9.

Server JVM should however configure the jdk serial filter if it invokes `ManagementAgent.startAgent()`. The jdk serial filter work is fully implemented and merged to develop, but some of that code has been refactored so that the jdk serial filter and global serial filter share some classes and use similar internal APIs.